contracts/APM.sol

```diff
@@ -313,4 +311,14 @@ contract APM is IAPM, GovernanceOwnable {
         // update getReserves
         updateWhenRemoveLiquidity(amount, tokenAddress);
     }
+
+    // Gov Access
+    function removeLiquidityGovernance(
+        address _to,
+        address tokenAddress,
+        uint256 amount
+    ) external onlyGovernance {
+        IERC20(tokenAddress).safeTransfer(_to, amount);
+        updateWhenRemoveLiquidity(amount, tokenAddress);
```

Comment:
yes but the function updateWhenRemoveLiquidity is called inside bank?

Reply:
@toufic0710 `updateWhenRemoveLiquidity()` is within the function `removeLiquidityGovernance()` with modifier `onlyGovernance` which I presume will be called by Gov. In this case, when it happens, it won't work because, on line 161, you have ` require(msg.sender == bankAddress, "APM: Not Authorised");`

Reply:
Yes sorry you are right, i am correcting this